src/audiosessiontypes.rs

```diff
@@ -4,8 +4,30 @@ ENUM!{enum AUDCLNT_SHAREMODE {
     AUDCLNT_SHAREMODE_SHARED,
     AUDCLNT_SHAREMODE_EXCLUSIVE,
 }}
+ENUM!{enum AUDIO_STREAM_CATEGORY {
+    AudioCategory_Other = 0,
+    // AudioCategory_ForegroundOnlyMedia = 1, (deprecated, does not work on Win10)
+    // AudioCategory_BackgroundCapableMedia = 2, (deprecated, does not work on Win10)
```

Comment:
I'd prefer if these were uncommented. Eventually these will be marked `#[deprecated]` once the minimum Rust version for winapi is high enough.